servicecontrol/capacity-and-planning.md

```diff
@@ -61,4 +61,4 @@ The transports supported by ServiceControl out-of-the-box (i.e. MSMQ, RabbitMQ,
 
 Azure Queues and Service Bus throughput varies significantly based on deployment options and multiple related variables inherent to cloud deployment scenarios.
 
-It is recommended that you plan and perform realistic throughput tests on ServiceControl using the transport of your choice and deployment options that are as close as possible to your planned production deployment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).
\ No newline at end of file
+We recommended that customers plan and perform realistic throughput tests on ServiceControl using the transport of your choice and deployment options that are as close as possible to your planned production deployment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).
```

Comment:
I can do some, I don't know what to say in response to just fix SC
On Tue, 9 Feb 2016 at 7:00 PM, Simon Cropp <notifications@github.com> wrote:

> In servicecontrol/capacity-and-planning.md
> <https://github.com/Particular/docs.particular.net/pull/1098#discussion_r52276331>
> :
>
> > \ No newline at end of file
> > +We recommended that customers plan and perform realistic throughput tests on ServiceControl using the transport of your choice and deployment options that are as close as possible to your planned production deployment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).
>
> @petersgiles <https://github.com/petersgiles> u going to address the
> above?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/1098/files#r52276331>
> .
>